recipes-multimedia/aamp/aamp_git.bb

```diff
@@ -30,6 +31,9 @@ PACKAGECONFIG ??= "\
     ${@bb.utils.contains('DISTRO_FEATURES', 'playready', 'playready', '', d)} \
 "
 PACKAGECONFIG[opencdm] = "\
+    -DCMAKE_WPEWEBKIT_JSBINDINGS=ON \
```

Comment:
Looking at Haseena feedback, I think this needs to be put on a separate PACKAGECONFIG flag